src/main/java/com/gmail/goosius/siegewar/SiegeWar.java

```diff
@@ -159,28 +159,28 @@ private void registerListeners() {
 
 	private void registerCommands() {
 		if(siegeWarPluginError) {
-			System.err.println(prefix + "SiegeWar is in safe mode. SiegeWar commands not registered");
+			severe("SiegeWar is in safe mode. SiegeWar commands not registered");
 		} else {
 			getCommand("siegewar").setExecutor(new SiegeWarCommand());
 			getCommand("siegewaradmin").setExecutor(new SiegeWarAdminCommand());
 		}
 	}
 
 	private void printSickASCIIArt() {
-		System.out.println("    _________.__                      ");
-		System.out.println("   /   _____/|__| ____   ____   ____  ");
-		System.out.println("   \\_____  \\ |  |/ __ \\ / ___\\_/ __ \\ ");
-		System.out.println("   /        \\|  \\  ___// /_/  >  ___/ ");
-		System.out.println("  /_______  /|__|\\___  >___  / \\___  >");
-		System.out.println("          \\/         \\/_____/      \\/ ");
-		System.out.println("       __      __                        ");
-		System.out.println("      /  \\    /  \\_____ _______          ");
-		System.out.println("      \\   \\/\\/   /\\__  \\\\_  __ \\         ");
-		System.out.println("       \\        /  / __ \\|  | \\/         ");
-		System.out.println("        \\__/\\  /  (____  /__|            ");
-		System.out.println("             \\/        \\/                ");
-		System.out.println("          By Goosius & LlmDl          ");
-		System.out.println("                                      ");
+		Bukkit.getLogger().info("    _________.__                      ");
+		Bukkit.getLogger().info("   /   _____/|__| ____   ____   ____  ");
+		Bukkit.getLogger().info("   \\_____  \\ |  |/ __ \\ / ___\\_/ __ \\ ");
+		Bukkit.getLogger().info("   /        \\|  \\  ___// /_/  >  ___/ ");
+		Bukkit.getLogger().info("  /_______  /|__|\\___  >___  / \\___  >");
+		Bukkit.getLogger().info("          \\/         \\/_____/      \\/ ");
+		Bukkit.getLogger().info("       __      __                        ");
+		Bukkit.getLogger().info("      /  \\    /  \\_____ _______          ");
+		Bukkit.getLogger().info("      \\   \\/\\/   /\\__  \\\\_  __ \\         ");
+		Bukkit.getLogger().info("       \\        /  / __ \\|  | \\/         ");
+		Bukkit.getLogger().info("        \\__/\\  /  (____  /__|            ");
+		Bukkit.getLogger().info("             \\/        \\/                ");
+		Bukkit.getLogger().info("          By Goosius & LlmDl          ");
+		Bukkit.getLogger().info("                                      ");

```

Comment:
:question: 
- Are the above calls mean to be Bukkit.getLogger().info()  instead of just info() ?

Reply:
This gets us an un-prefixed line in the log, making the ascii nicer.